### PR TITLE
Create 12_how-do-i-change-my-avatar.md

### PR DESCRIPTION
**Pull request by DarklingMuse**

### CODE REVIEW

File: 03_pictures-videos-and-writings/12_how-do-i-change-my-avatar.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#How do I change my avatar?
```
Comment (nielsenramon) [inline 03_pictures-videos-and-writings/12_how-do-i-change-my-avatar.md:1]: Missing space after the `#`

File: 03_pictures-videos-and-writings/12_how-do-i-change-my-avatar.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+You can use a picture that you have already uploaded as be your avatar (profile picture) at anytime by:
```
Comment (nielsenramon) [inline 03_pictures-videos-and-writings/12_how-do-i-change-my-avatar.md:3]: This reads very weird! :D I would simplify!

`You can use any picture you've uploaded to be your avatar by:`

File: 03_pictures-videos-and-writings/12_how-do-i-change-my-avatar.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+1. Visiting the "Options" menu link located beneath the caption of the picture or video that you want to use as your avatar
```
Comment (nielsenramon) [inline 03_pictures-videos-and-writings/12_how-do-i-change-my-avatar.md:5]: You can't put a video as avatar :D

File: 03_pictures-videos-and-writings/12_how-do-i-change-my-avatar.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+2. Selecting “Set as Avatar” to make the picture your main profile picture
+
+Alternatively you can upload a new picture to use as your avatar by [following these steps](01_upload-picture-profile.md)
```
Comment (nielsenramon) [inline 03_pictures-videos-and-writings/12_how-do-i-change-my-avatar.md:8]: Will need to be a direct link I'm afraid!

**Comment by sylee-** [comment]
@nielsenramon - Jil updated the article for this a few days ago.  #104 Are we changing that one or creating a new one?

**Comment by nielsenramon** [comment]
@sylee- Oh lol let's not bother changing it again then please.

**Pull request closed without merging**